testing/download-mongodb.ts

```diff
@@ -64,12 +67,21 @@ async function lookupDownloadUrl(versionInfo: VersionInfo): Promise<string> {
   return downloadInfo.archive.url;
 }
 
+const downloadPromises: Record<string, Promise<string>> = {};
 // Download mongod + mongos and return the path to a directory containing them.
-export async function downloadMongoDb() {
-  const resp = await fetch('https://downloads.mongodb.org/full.json');
-  const json: FullJSON = await resp.json();
-  const productionVersions = json.versions
-    .filter((version: VersionInfo) => version.production_release)
+export async function downloadMongoDb(targetVersion = '*'): Promise<string> {
```

Comment:
`'*'` why not `latest`?

Reply:
because `latest` is not a meaningful semver string … I’ll update the variable name to reflect that this uses semver :)